configuration/concepts/breast_examination.csv

```diff
@@ -0,0 +1,23 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:800
+0c510fac-1318-4e27-8eab-27cbb3a548ae,,,Results,Results,Resultados,Resultados,,Misc,Text,,,,,,,,,,,
+0fa92108-224f-472c-b4de-3a9bf1743d21,,,Dates,Dates,fechas,fechas,,Misc,Text,,,,,,,,,,,
+e3a3ff33-583f-4a4e-b2ab-3e95ccbb24f2,,,Cyclic,Cyclic,Cíclico,Cíclico,,Misc,N/A,,,,,,,,,,,
+e21bc895-e3c5-429f-85cb-2c9524edd0fa,,,Non Cyclic,Non Cyclic,No cíclico,No cíclico,,Misc,N/A,,,,,,,,,,,
+c23d4502-1798-42ad-9cb6-e35d94e782a9,,,Breast Lump,Breast Lump,Bulto en el pecho,Bulto en el pecho,,Misc,Coded,,Yes; No,,,,,,,,,
+57dfa129-bcfe-456e-9ef6-5c53925becad,,,Breast Pain,Breast Pain,Dolor en los senos,Dolor en los senos,,Misc,Coded,,Cyclical; Non-Cyclical,,,,,,,,,
+77e70e54-f636-4f3c-b41a-8ea923d0ee35,,,Discharge,Discharge,Descarga,Descarga,,Misc,Coded,,Yes; No,,,,,,,,,
+de1cfa19-364b-4ed7-bef8-017ae68143f5,,,Inversion,Inversion,inversión,inversión,,Misc,Coded,,Yes; No,,,,,,,,,
+3e900c6c-964e-42b7-b4fe-d24ca2fa729e,,,Rash,Rash,Erupción,Erupción,,Misc,Coded,,Yes; No,,,,,,,,,
+4f9af643-0697-47cb-952f-6b2e29482747,,,Nipple Abnormalities,Nipple Abnormalities,Anormalidades del pezón,Anormalidades del pezón,,ConvSet,N/A,,,Discharge; Inversion; Rash,,,,,,,,
+f19a46f6-0b70-48c5-afa5-98bee87a36aeO,,,Oral Contraceptive Pills,Oral Contraceptive Pills,Píldoras anticonceptivas orales,Píldoras anticonceptivas orales,,Misc,Coded,,Yes; No,,,,,,,,,
+01edf3d3-a3dc-4174-9847-dfb5ffdbef9d,,,Hormone Replacement Therapy:,Hormone Replacement Therapy:,Terapia de reemplazamiento de hormonas:,Terapia de reemplazamiento de hormonas:,,Misc,Coded,,Yes; No,,,,,,,,,
+8f0ec4ef-8f95-4db1-9e88-ab154d92912a,,,Mammogram,Mammogram,Mamografía,Mamografía,,Misc,N/A,,,,,,,,,,,
+d6983895-4175-4676-a505-6f7ecb4b0ead,,,Ultrasound,Ultrasound,Ultrasonido,Ultrasonido,,Misc,N/A,,,,,,,,,,,
+5deb227d-1142-4117-8743-8773d7a1dbf3,,,Medications,Medications,medicamentos,medicamentos,,Misc,Text,,,,,,,,,,,
+8868de4a-9011-4afe-ab7e-02dd9962d37d,,,Allergies,Allergies,alergias,alergias,,Misc,Text,,,,,,,,,,,
+adca267e-a622-4b2d-bd92-e68dad4abdc6,,,History of Breast Issues,History of Breast Issues,Historia de problemas de mama,Historia de problemas de mama,,Misc,Coded,,Yes; No,,,,,,,,,
+db4ce47b-052d-4c90-892f-bb7f13115e8e,,,Seen previously by MD,Seen previously by MD,Visto anteriormente por MD,Visto anteriormente por MD,,Misc,Coded,,Yes; No,,,,,,,,,
+ff0db523-f7b5-4d36-95cf-a25a74672fc0,,,Diagnosis,Diagnosis,Diagnóstico,Diagnóstico,,Misc,Text,,,,,,,,,,,
+f5b35153-76f7-468c-bc39-f44c942f154b,,,Previous Treatment,Previous Treatment,Tratamiento previo,Tratamiento previo,,Misc,Text,,,,,,,,,,,
+5fa09c90-a5ff-484f-bf41-3f8bd4faa901,,,Other,Other,Otro,Otro,,Misc,Text,,,,,,,,,,,
```

Comment:
Are we using this concept anywhere? We shall use 'Details' concepts to capture results